portfolio/src/main/webapp/index.html

```diff
@@ -5,18 +5,58 @@
     <title>Lorem</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
   </head>
   <body>
+    <br/>
+    <br/>
+    <ul class="icon" >
+    <li><a class="long-icon"  href="/login"><img src="./images/loginicon.png" height="100" width="200"></a></li>
+    <li><a  href="/journal.html">&nbsp;&nbsp;<img src="./images/journalicon.png" height="120" width="120"></a></li>
+    <li><a  href="/charts.html">&nbsp;&nbsp;<img src="./images/chartsicon.png" height="120" width="120"></a></li>
+    <li><a  href="/resources.html">&nbsp;&nbsp;<img src="./images/resourcesicon.png" height="120" width="120"></a></li>
+    <li><a  href="/index.html">&nbsp;&nbsp;<img src="./images/homeicon.png" height="150" width="120"></a></li>
+    </ul>
+
+    <ul class="lorem-logo">
+    <li>&ensp;L</li>
+    <li>&nbsp;O</li>
+    <li>&ensp;R</li>
+    <li>&ensp;E</li>
+    <li>&nbsp;M</li>
+    </ul>
+    <!-- TODO: Make a description box of feature appear when 
+               user hovers over navigation link. enter. scribe. track. discover. begin.-->
+    <!-- TODO: Change icon images to "icon-selected" images on hover-->
+    <!--TODO: move buttons/text between margins so that functionality is accessible -->
+
+     
     <div id="content">
-      <h1>Welcome to Lorem!</h1>
-      <p>Login here.</p>
       <button onclick="revealLogin()" id="myButton" class="float-left submit-button" >Login</button>
       <div id="greeting-container"></div>
-      <p>Click <a href="/login">here</a> here to login.</p>
-      <p>Click <a href="/journal.html">here</a> to go to the journal.</p>
-      <p>Click <a href="/charts.html">here</a> to go to the mood trackers.</p>
-      <p>Click <a href="/resources.html">here</a> to see the resources page.</p>
 
-    </div>
-  </body>
+    <script>
+        var currentTime = new Date().getHours();
+        if (currentTime <= 5 | currentTime>20) {
+            if (document.body) {
+                document.body.style.backgroundImage = "url('https://4.bp.blogspot.com/-vEBoI0UVq5I/XxOltUqiyQI/AAAAAAAAAm8/Vci6MSP-Cw8DGvMsmQCPYkoH_J-tykfQwCLcBGAsYHQ/s1600/nigh.png')";
+            }
+        }
```

Comment:
Move this JS code to a shared file.  That way you can choose to use this on multiple pages